chain/jsonrpc/src/lib.rs

```diff
@@ -328,16 +332,55 @@ impl JsonRpcHandler {
         match self.client_addr.send(Status { is_health_check: true }).await {
             Ok(Ok(_)) => Ok(Value::Null),
             Ok(Err(err)) => Err(RpcError::new(-32_001, err, None)),
-            Err(_) => Err(RpcError::server_error::<String>(None)),
+            Err(_) => Err(RpcError::server_error::<()>(None)),
         }
     }
 
     pub async fn status(&self) -> Result<Value, RpcError> {
         match self.client_addr.send(Status { is_health_check: false }).await {
             Ok(Ok(result)) => jsonify(Ok(Ok(result))),
             Ok(Err(err)) => Err(RpcError::new(-32_001, err, None)),
-            Err(_) => Err(RpcError::server_error::<String>(None)),
+            Err(_) => Err(RpcError::server_error::<()>(None)),
+        }
+    }
+
+    /// Expose Genesis Config (with internal Runtime Config) without state records to keep the
+    /// output at a reasonable size.
+    ///
+    /// See also `genesis_records` API.
+    pub async fn genesis_config(&self) -> Result<Value, RpcError> {
```

Comment:
Since you already created `GenesisRecordsView`, why not create `GenesisConfigView` so that you don't have to mangle the json object?

Reply:
@bowenwang1996 I would then need to manually map a whole lot of various fields :disappointed:

Reply:
Just for the record, I cannot use `GenesisConfig` structure as it has `records` field required. I am currently resolving the merge conflict and considering to solve this one way or another as `write_to_config_and_records_files` also plays and dances around decoupling the records.

@bowenwang1996 @nearmax Do you think it is fine to transform `GenesisConfig` into: 

```rust
Genesis {
    config: GenesisConfig,
    records: GenesisRecords,
}
```

?

Reply:
I think that's a great naming and idea.

Reply:
sounds good